src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -606,6 +606,15 @@ trait EventFormats {
       "eventType" -> change.eventType
     )
   }
+  implicit lazy val UnknownInstanceTerminatedEventWrites: Writes[UnknownInstanceTerminated] = Writes { change =>
+    Json.obj(
+      "instanceId" -> change.id,
+      "runSpecId" -> change.runSpecId,
+      "instanceStatus" -> change.status.toString,
```

Comment:
We don't have that at hand when the task is unknown, we have only the mesos TaskStatus and that doesn't provide the runSpec version safely (there might be a label iirc)